package.json

```diff
@@ -17,7 +18,9 @@
     "package": "cross-env NODE_ENV=production node -r babel-register -r babel-polyfill webpack/package.js",
     "package-all": "npm run package -- --all",
     "postinstall": "node node_modules/fbjs-scripts/node/check-dev-engines.js package.json",
+    "preinstall": "which electron || npm install electron@1.4.12",
```

Comment:
If you want to use system `electron` executable, npm doesn't allow for a way to specify that. Sadly we need this when using Nix, since electron is provided  by it.

The only way is to optionally install it if `electron` is missing.

Maybe I've missed some better method while googling :)